source/manual/business-readiness-finder.html.md

```diff
@@ -0,0 +1,61 @@
+---
+title: EU exit business readiness finder
+section: Business readiness finder
+layout: manual_layout
+parent: "/manual.html"
+owner_slack: "#govuk-euexit-ready"
+last_reviewed_on: 2019-06-19
+review_in: 6 months
+---
+
+The [EU exit business readiness finder](https://www.gov.uk/find-eu-exit-guidance-business) is a special finder built to help businesses prepare for EU exit. A number of proprietary modifications have been made to the normal [finder-frontend](/apps/finder-frontend.html) model, which we've documented here.
+
+For a list of all of the YAML files that drive the finder and its related pages, see: [Publish changes to the business readiness finder](/manual/business-readiness-publish-changes.html).
+
+## Overview
+
+The full journey for the business readiness finder is made up of the following content items:
+
+1. The start page: [`/business-uk-leaving-eu`](https://www.gov.uk/business-uk-leaving-eu)
+2. The Q&A: [`/prepare-business-uk-leaving-eu`](https://www.gov.uk/prepare-business-uk-leaving-eu)
+3. The finder: [`/find-eu-exit-guidance-business`](https://www.gov.uk/find-eu-exit-guidance-business)
+4. The email signup page: [`/find-eu-exit-guidance-business/email-signup`](https://www.gov.uk/find-eu-exit-guidance-business/email-signup)
+
+Editors can request changes to the finder via a [form](https://support.integration.publishing.service.gov.uk/eu_exit_business_readiness_request/new) in the [Support](https://github.com/alphagov/support) app.
+
+### Start page
+
+A normal page which can be updated in Mainstream Publisher.
+
+### Q&A
+
+The Q&A is [documented in finder-frontend](https://github.com/alphagov/finder-frontend/blob/master/docs/q-and-a.md).
+
+### Finder
+
+#### Facets
+
+The finder uses the search-api to find and filter results. Whilst standard finders get more granular as you select values in different facets (the number of results goes down), the business finder gets more _broad_ (number of results goes up).
+
+Standard finders retrieve content published by a single publishing app, where the ability to add facets is built directly into the publishing interface. The result is facets are stored under `details.facets` in the [finder's content item](https://www.gov.uk/api/content/cma-cases). Think of `details` as containing the body of the content.
+
+The business readiness finder collates content from _multiple_ publishing apps, not all of which have facets built into their interface. However, content-tagger provides a central way of tagging to links (which themselves come from the Publishing API), so the business finder uses content tagger to avoid having to add facets to the interface of every relevant publishing app. The [business finder content item](https://www.gov.uk/api/content/find-eu-exit-guidance-business) therefore has facet information under `links.facet_group`. Think of `links` as containing metadata describing the content.
```

Comment:
I'd remove this "Think of `links` as containing metadata describing the content." Links are more complicated than this sentence implies. If you wanted, you could just link to the docs in publishing-api: https://github.com/alphagov/publishing-api/blob/master/doc/link-expansion.md#patch-link-set---link-set-links